_build/css/main.css

```diff
@@ -170,6 +170,7 @@ h4 {
 
 .label {
   font-weight: 400;
+  margin-top: 8px;
```

Comment:
Should we lower "h3 margin-bottom" to 8px as the gap between title and labels is quite big now? H3 is not used that much and I think it's also okay to be 8px on the other spots that would be affected.

Reply:
Discussed via chat - this needs a bit better implementation, but other fixes are needed as soon as possible, so this will be done separately